### PR TITLE
Update Windows project to fix compilation and add auto-linking

### PR DESCRIPTION
**Pull request by tritao**
Should fix https://github.com/react-native-progress-view/progress-view/issues/50

### CODE REVIEW

File: windows/progress-view/progress-view.vcxproj [hunk 1]
```diff
@@ -162,18 +161,25 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
+  <ImportGroup Label="ReactNativeWindowsTargets">
```
Comment (jonthysell) [inline windows/progress-view/progress-view.vcxproj:164]: Adding the import of the CppLib.targets is correct, but you also need to add the import for the CppLib.props earlier in the file, and remove the project references for Microsoft.ReactNative and Microsoft.ReactNative.Cxx.

Comment (jonthysell) [reply windows/progress-view/progress-view.vcxproj:164]: You can see similar changes I made for react-native-video's project here: https://github.com/react-native-video/react-native-video/pull/2206/files#diff-21a98c9b66b676799e297b1bf6028f32087df1baf8ebbc109181f75e8348a4e6

Comment (tritao) [reply windows/progress-view/progress-view.vcxproj:164]: Thanks for the feedback, I'll update the PR tomorrow.

**Comment by tritao** [comment]
@Naturalclar You seem to have merged the last few PRs, could you take a look at this one?

**Review by asklar — APPROVED** [review]
LGTM, ping @jonthysell in case he has any concerns.

**Review by Naturalclar — APPROVED** [review]
@asklar thanks for the review!

**Comment by Naturalclar** [comment]
@tritao the change has been published in v1.2.4 :tada: thanks for the change!

**Pull request merged**